### DISCUSSION TITLE
Interaction between the rendered / preview and source in RStudio

### DISCUSSION THREAD

**Discussion by olivroy** [quarto-dev/quarto-web#215] [General]
Provide an interface similar to Overleaf, that allows to click back and forth between editor and rendered output 

i.e. When double click in the pdf in Overleaf, it takes you to the right source location. It facilitates editing.

**Comment by jjallaire** [comment]
Improving the authoring experience is definitely a high priority (including making it easier to go between editor and rendered doc as you suggest).
